### PR TITLE
Dev/Projects page

### PR DESCRIPTION
**Pull request by KushalD1999**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -99,20 +105,59 @@ <h1>Skills</h1>
...
+            <div class="project-section">
+               <h3 id = "projects-headline">Projects </h3>
+               <div class="project-card panel panel-default">
+                  <div class="panel-heading">
+                     <h3 class="panel-title">EasyRead (Microsoft Discover AI Challenge)</h3>
+                  </div>
+                  <div class="panel-body">
+                     <ul>
+                        <li class="project-desc">Our team was selected in Top 6 for Microsoft’s Discover AI Challenge where over 200 teams came up with an innovative solution to solve a sustainability problem using artificial intelligence technology</li>
+                        <li class="project-desc"> <strong>Purpose of app: </strong>Using the Tesseract Optical Character Recognition library , our aim was to aid people suffering from dyslexia and other reading disabilities.</li>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:129]: Nit: extra space before ,

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -99,20 +105,59 @@ <h1>Skills</h1>
...
+                  <a class="fa fa-chevron-down fa-4x" href="#projects"></a>
+               </div>
             </div>
          </section>
          <section id="projects">
+            <div class="project-section">
+               <h3 id = "projects-headline">Projects </h3>
+               <div class="project-card panel panel-default">
+                  <div class="panel-heading">
+                     <h3 class="panel-title">EasyRead (Microsoft Discover AI Challenge)</h3>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:124]: I would love to see links to the projects (if possible).  Github repo links, competition site, etc.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -99,20 +105,59 @@ <h1>Skills</h1>
...
+                     <h3 class="panel-title">Movie Database System</h3>
+                  </div>
+                  <div class="panel-body">
+                     <ul>
+                        <li class="project-desc">Created an Online Movie Databse using Node.jS and Express.JS</li>
+                        <li class="project-desc">Implemented a platform that returns top 10 movies and TV shows based on the Search Query by calling an API endpoint</li>
+                     </ul>
+                  </div>
+               </div>
+               <p id="project-endline">Check out my more projects on Github. <br>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:156]: Grammar: Check out more of my projects

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -49,13 +52,13 @@ section {
...
-    text-align: center;
-
+	width: 100%;
+	text-align: center;
 }
 
 .fa {
-	padding: 15px 15px;;
+	padding: 15px 15px;
+	;
```
Comment (mctavish) [inline portfolio/src/main/webapp/style.css:61]: Delete this

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (KushalD1999) [reply portfolio/src/main/webapp/index.html:124]: I will add them

**Pull request merged**